test/functional/test_framework/test_node.py

```diff
@@ -57,9 +57,11 @@ def __init__(self, i, dirname, extra_args, rpchost, timewait, binary, stderr, mo
             self.binary = binary
         self.stderr = stderr
         self.coverage_dir = coverage_dir
-        # Most callers will just need to add extra args to the standard list below. For those callers that need more flexibity, they can just set the args property directly.
+        # Most callers will just need to add extra args to the standard list below.
+        # For those callers that need more flexibity, they can just set the args property directly.
+        # Note that common args are set in the config file (see initialize_datadir)
         self.extra_args = extra_args
-        self.args = [self.binary, "-datadir=" + self.datadir, "-server", "-keypool=1", "-discover=0", "-rest", "-logtimemicros", "-debug", "-debugexclude=libevent", "-debugexclude=leveldb", "-mocktime=" + str(mocktime), "-uacomment=testnode%d" % i]
+        self.args = [self.binary, "-datadir=" + self.datadir, "-logtimemicros", "-debug", "-debugexclude=libevent", "-debugexclude=leveldb", "-mocktime=" + str(mocktime), "-uacomment=testnode%d" % i]
```

Comment:
BTW, why not move `-logtimemicros` and `-debug*` to conf (all static parameters really)?

Reply:
I think when you later spin up a node for debugging the datadir (block state, wallet state), you don't need to log micro time stamps. Also, I am not sure how the debug flags interfere when the conf sets `-debug` and the command line `-nodebug` ... Better leave it up for the user to set those explicitly, I think?

Reply:
That's reasonable, thanks.